flags/1x1/ws.svg

```diff
@@ -1,7 +1,7 @@
 <svg xmlns="http://www.w3.org/2000/svg" id="flag-icon-css-ws" width="512" height="512">
   <g fill-rule="evenodd" stroke-width="1pt">
-    <path fill="red" d="M0 0h512v512H0z"/>
-    <path fill="#00006b" d="M0-1h256v128H0z"/>
-    <path fill="#fff" d="M132.5 116.6l-10.1-6.9-9.8 7 3.1-12.2-9.3-7.6 12-.7 4-11.6 4.4 11.8 11.8.3-9.4 8zm-1.7-81.3l-8.1-5.2-8 5.2 2.2-9.8-7.1-6.3 9.5-.8 3.5-9.1 3.7 9.2 9.3.7-7.2 6.5zM94.9 64.6L87 59l-8 5.4 2.4-9.5-7.4-6.3 9.4-.4 3.5-9.4 3.4 9.2 9.5.6-7.4 6.2zm72-6.6l-7.6-4.9-7.4 5 2.1-9-6.8-5.8 9-.7 3.1-8.5 3.4 8.6 8.7.5-6.8 6zm-21.3 21.4l-5-3.2-5 3.2 1.3-6-4.5-4 5.9-.5 2.3-5.7 2.3 5.7 5.8.5-4.4 4z"/>
+    <path fill="#ce1126" d="m0 0h512v512h-512z"/>
+    <path fill="#002b7f" d="m0-1h329.76v262.51h-329.76z"/>
+    <path fill="#fff" d="m185.38 242.35-21.123-14.431-20.496 14.64 6.4834-25.515-19.45-15.895 25.097-1.464 8.3657-24.261 9.2023 24.679 24.679 0.62742-19.659 16.731zm-3.5554-170.03-16.941-10.875-16.731 10.875 4.6011-20.496-14.849-13.176 19.869-1.6731 7.32-19.032 7.7383 19.241 19.45 1.464-15.058 13.594zm-75.082 61.279-16.522-11.712-16.731 11.294 5.0194-19.869-15.477-13.176 19.659-0.83657 7.32-19.659 7.1108 19.241 19.869 1.2549-15.477 12.967zm150.58-13.803-15.895-10.248-15.477 10.457 4.392-18.823-14.222-12.13 18.823-1.464 6.4834-17.777 7.1108 17.986 18.195 1.0457-14.222 12.549zm-44.547 44.757-10.457-6.6926-10.457 6.6926 2.7188-12.549-9.4114-8.3657 12.339-1.0457 4.8103-11.921 4.8103 11.921 12.13 1.0457-9.2023 8.3657z"/>
```

Comment:
Sorry for being picky.. but can you make the blue area to occupy here again a quarter of the whole area? (half the width and half the height)